examples_extra/Placeholders/Sample/PostNode.m

```diff
@@ -60,6 +60,10 @@ - (instancetype)init
 - (UIImage *)placeholderImage
 {
   CGSize size = self.calculatedSize;
+  if (CGSizeEqualToSize(size, CGSizeZero)) {
+    return nil;
+  }
+  
   UIGraphicsBeginImageContext(size);
   [[UIColor colorWithWhite:0.9 alpha:1] setFill];
   UIRectFill((CGRect){CGPointZero, size});
```

Comment:
Hmm, this is very inefficient.

We should consider bringing back placeholderColor, since no backing store would be created.  But until then, we should use a 1px stretchable image for huge memory / drawing time savings.  Then it technically wouldn't even need to check for zero size.